.github/pull_request_template.md

```diff
@@ -1,18 +1,22 @@
-## What is the purpose of the change
+## Description
 
-_Briefly describe the change you are making, this helps the reviewer by providing an overview._
-
-## Are there related marketing activities
-
-_Are there any marketing activities related to the related feature or component? If so, please briefly describe them._
+<!-- This helps the reviewers by providing an overview of what to expect in the PR. Linking to an issue is preferred but not required. -->
 
 ## When should this change go live?
 
-_Does this change need to be released before a certain date or milestone? If so, when?_
+<!-- PRs merged go to stage.docs.camunda.io first and must be manually released to docs.camunda.io. -->
+<!-- Help the DevEx team prioritize our work (reviews, merges, etc.) by opening PRs sooner. -->
+
+- [ ] There is no urgency with this change.
+- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
+- [ ] This functionality is already available but undocumented.
+- [ ] This is a bug fix or security concern.
```

Comment:
Maybe something along the lines of "This change is not yet live and should not be merged until X (apply hold label)"?

Reply:
Added!